lib/emque/consuming/adapters/rabbit_mq.rb

```diff
@@ -15,6 +15,7 @@ def self.load
           require_relative "rabbit_mq/manager"
           require_relative "rabbit_mq/worker"
           require_relative "rabbit_mq/retry_worker"
```

Comment:
wonder if we should rename this to error worker to avoid confusion of retries from delayed_messages queue and retries from error queue?